components/shared/hooks/UseMediaQuery/Example.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
i think we dont need a story for hook.
Stories are only for components, right?
WDYT?

Reply:
But how would other people know about them if not in the docs?

Reply:
I've same doubt

Reply:
Why not both?